.github/workflows/integration.yml

```diff
@@ -6,6 +6,10 @@ on:
     branches:
       - master
   pull_request:
```

Comment:
Just to avoid forgetting...

Before merging, this needs to be changed to `pull_request_target`. And all the other references need to be updated accordingly